Gems/PhysX/Code/Include/PhysX/EditorColliderComponentRequestBus.h

```diff
@@ -83,10 +83,13 @@ namespace PhysX
 
     using EditorColliderComponentRequestBus = AZ::EBus<EditorColliderComponentRequests>;
 
+    /// <EditorColliderValidationRequests>
+    /// This is a Bus in order to communicate the status of the meshes of the collider and avoid dependencies with the rigidbody
+    /// </EditorColliderValidationRequests>
     class EditorColliderValidationRequests : public AZ::ComponentBus
     {
     public:
-        // TODO
+        /// Checks if the the mesh in the collider is correct with the current state of the Rigidbody!
```

Comment:
nit: Please don't add exclamation marks to comments, never want them to seem shouty 😝